website/files/models/github.py

```diff
@@ -14,3 +14,7 @@ class GithubFolder(GithubFileNode, Folder):
 
 class GithubFile(GithubFileNode, File):
     version_identifier = 'ref'
+
+    def touch(self, auth_header, revision=None, ref=None, branch=None, **kwargs):
+        revision = revision or ref or branch
+        return super(GithubFileNode, self).touch(auth_header, revision=revision, **kwargs)
```

Comment:
Is there a reason why the super call isn't `super(GithubFile, self).touch(...)`?

This call as written would skip any method implemented on the direct parent class. (it instead calls the method on the grandparent, aka the superclass of the parent)